File: stephanie/Problema0231/Problema0231.js
var popA=90000, popB=300000 ;
var i=0;

while (popB > popA){
    i = i + 1;
    popA = popA + popA*3/100;
    popB = popB + popB*1.2/100;
}
console.log("A população da cidade A ultrapassará a população da "+"cidade B daqui a "+i+" anos.");

File: stephanie/Problema0231/Problema0231.py
popA=90000
popB=300000 
i=0
while (popB > popA):
   i = i + 1
   popA = popA + popA*3/100
   popB = popB + popB*1.2/100
print("A população da cidade A ultrapassará a população da ","cidade B daqui a ",i," anos.")
